stl/inc/regex

```diff
@@ -1257,9 +1257,9 @@ struct _Buf { // character buffer
         return _Chrs;
     }
 
-    void _Insert(_Elem _Ch) { // append _Ch
+    void _Insert2(_Elem _Ch) { // append _Ch

```

Comment:
I think MSVC STL prefers `_v2` suffix over `2`. A plain `2` seemingly introduces some semantic implication, although it should not in this case.

Reply:
I don't mind either way, but the `_v2` suffix is not the style that has been used in `<regex>` for new versions up till now. See Casey's #131 and all the other occurrences of functions ending in `1` or `2` in this file (although I have to admit I'm not following this precedent strictly as well because I find it weird to label the second version of a function by appending `1`). On the other hand, there are no functions ending in `_vxxx`.

Reply:
We've got examples of both, e.g. `_Stodx_v3` and `_Orphan_all_unlocked_v3`. I'm fine with either.